registry/server/migrations/20201027180112_settings_globalSpinner.ts

```diff
@@ -0,0 +1,30 @@
+import * as Knex from "knex";
+
+
+export async function up(knex: Knex): Promise<any> {
+    await knex('settings').insert([{
+        key: 'globalSpinner.enabled',
```

Comment:
I guess we should use here keys from SettingKeys and the same about Scopes and SettingTypes

Reply:
fixed